cli/tools/registry/pm/outdated.rs

```diff
@@ -147,7 +166,13 @@ fn print_outdated(
 
   if !outdated.is_empty() {
     outdated.sort();
-    print_outdated_table(&outdated);
+    match output_fmt {
+      OutdatedOutputFmt::Table => print_outdated_table(&outdated),
+      OutdatedOutputFmt::Json => {
+        let json = serde_json::to_string_pretty(&outdated)?;
```

Comment:
If this feature is added, we should use object notation and then include a version number (ex. "1"), similar to the other `--json` output. That allows adding new properties in the future in a backwards compatible way and also attaches a version to the object.

Reply:
+1. Let's hammer out the format first (see above) and then we can add a version field. Typically semantic version I guess (like 1.0.0)?

Reply:
Just a single integer is fine. Additions fall under the same version, but breaking changes cause a version bump. Otherwise people need to parse the version with something like `std/semver` to see if they can understand it rather than just doing an equality check.